gateleen-kafka/src/test/java/org/swisspush/gateleen/kafka/KafkaHandlerTest.java

```diff
@@ -128,15 +129,15 @@ public void initWithWildcardConfigResource(TestContext context) {
         context.assertFalse(handler.isInitialized());
 
         handler.initialize().setHandler(event -> {
-            verify(repository, times(2)).closeAll();
-
+            // depending whether resourceChanged fires or not we get one or two invocations
```

Comment:
not related to the PR, the test sometimes fails on travis
@geekrumper can you kindly have a look if this makes sense / propose changes as needed, thx.

Reply:
@dominik-cnx I think this is okay, if we can't guarantee that it always triggers two times.